<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Events</class>
 <widget class="QWidget" name="Events">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabBox">
    <attribute name="title">
     <string>事件列表</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>740</width>
       <height>491</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <column>
      <property name="text">
       <string>事件名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>结束时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>提醒时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>详细描述</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>是否完成</string>
      </property>
     </column>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>510</y>
       <width>571</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushBtnAdd">
        <property name="text">
         <string>添加</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushBtnModify">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushBtnDel">
        <property name="text">
         <string>删除</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/resources/del.png</normaloff>:/resources/del.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tabToday">
    <attribute name="title">
     <string>今日任务</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetToday">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>740</width>
       <height>490</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <column>
      <property name="text">
       <string>事件名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>开始时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>结束时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>提醒时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>详细描述</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>是否完成</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
